website/docs/r/api_management_gateway_api.html.markdown

```diff
@@ -0,0 +1,75 @@
+---
+subcategory: "API Management"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_api_management_gateway_api"
+description: |-
+  Manages a API Management Gateway API.
+---
+
+# azurerm_api_management_gateway_api
+
+Manages a API Management Gateway API.
+
+## Example Usage
+
+```hcl
+data "azurerm_api_management" "example" {
+  name                = "example-api"
+  resource_group_name = "example-resources"
+}
+
+data "azurerm_api_management_api" "example" {
+  name                = "search-api"
+  api_management_name = data.azurerm_api_management.example.name
+  resource_group_name = data.azurerm_api_management.example.resource_group_name
+  revision            = "2"
+}
+
+data "azurerm_api_management_gateway" "example" {
+  gateway_id          = "my-gateway"
+  api_management_name = data.azurerm_api_management.example.name
+  resource_group_name = data.azurerm_api_management.example.resource_group_name
+}
+
+resource "azurerm_api_management_gateway_api" "example" {
+  resource_group_name = data.azurerm_api_management.example.resource_group_name
+  api_name            = data.azurerm_api_management_api.example.name
+  gateway_id          = data.azurerm_api_management_product.example.gateway_id
+  api_management_name = data.azurerm_api_management.example.name
```

Comment:
Can you update this example for the new properties?

Reply:
updated